stacks/dotnet80/devfile.yaml

```
@@ -0,0 +1,57 @@
+schemaVersion: 2.2.0
+metadata:
+  name: dotnet80
+  displayName: .NET 8.0
+  description: .NET 8.0 application
+  icon: https://github.com/dotnet/brand/raw/main/logo/dotnet-logo.png
+  tags:
+    - .NET
+    - .NET 8.0
+  projectType: dotnet
+  language: .NET
+  version: 1.0.3
```

```suggestion
  version: 1.0.0
```

Since this is an entirely new stack I am inclined to say we should start the versioning to reflect that.